eventuals/static-thread-pool.h

```diff
@@ -520,6 +534,7 @@ struct _StaticThreadPoolSchedule {
           std::move(k),
           pool_,
           requirements_,
+          std::move(requirements_->name),
```

Comment:
Some folks might want to reuse a requirements, so I think we should properly propagate a name all the call to create the `Composable`.